src/jetson/models/SSD/data/__init__.py

```diff
@@ -0,0 +1 @@
+from .config import *
```

Comment:
Generally using the syntax `from mymodule import *` is frowned upon since it is unclear what is being exposed. This can lead to unexpected behavior when adding new features or making changes later on.